package/firmware/ipq-wifi/Makefile

```diff
@@ -76,7 +77,7 @@ define ipq-wifi-install-one
     $(call ipq-wifi-install-one-to,$(1),$(2),QCA9984/hw1.0),\
   $(if $(filter $(suffix $(1)),.QCA99X0 .qca99x0),\
     $(call ipq-wifi-install-one-to,$(1),$(2),QCA99X0/hw2.0),\
-  $(if $(filter $(suffix $(1)),.IPQ8074 .ipq8074),\
+  $(if $(filter $(suffix $(1)),.IPQ8074 .ipq8074 .ipq8174),\
```

Comment:
I am following your PR changes and into forked repo. https://github.com/i-srinivas/openwrt/tree/rbr750 that I am working on spare time. 

Found the above and it seems - the code filters for spell cases. Do you think based on the above we also need to add upper case '.IPQ8174' here ?

Reply:
Our file is in lower case, but yeah we could add the upper case name too

Reply:
ok, lets keep this consistent - and helpful for newbies like me.

Reply:
Honestly for the sake of it I would keep ipq8074... @robimarko any problem with that?

Reply:
FCC details shows IPQ8174, I do not know the intricacies between chip differences - it would be good to keep the chip # to avoid any confusion for people reading the code 

https://fccid.io/PY319400470/Internal-Photos/Internal-photos-4603355.pdf

Reply:
Those bdf extension are totally arbitrarily. (In fact if we should be precise there should be separation for ipq8072)
The extension is really to differentiate one family of BDF to another. The idea is to not be that precise about the bdf since they are all part of a bigger family.